SmallStatusWidget/SmallStatusWidget.swift

```diff
@@ -0,0 +1,239 @@
+            var eventualGlucoseString: String?
+            if let unit = unit, let eventualGlucose = eventualGlucose {
+                let glucoseFormatter = NumberFormatter.glucoseFormatter(for: unit)
+                eventualGlucoseString = glucoseFormatter.string(from: eventualGlucose.quantity.doubleValue(for: unit, withRounding: true))
+            }
+            
+            completion(SmallStatusEntry(
+                date: Date(),
+                lastLoopCompleted: lastCompleted,
+                closeLoop: closeLoop,
+                currentGlucose: currentGlucose,
+                previousGlucose: previousGlucose,
+                unit: unit,
+                sensor: context.glucoseDisplay,
+                netBasal: netBasal,
+                eventualGlucose: eventualGlucoseString,
+                minsAgo: 0
+            ))
+        }
+    }
+}
+
+
+struct SmallStatusEntry: TimelineEntry {
+    var date: Date
+    
+    let lastLoopCompleted: Date?
+    let closeLoop: Bool
+    
+    let currentGlucose: GlucoseValue?
+    let previousGlucose: GlucoseValue?
+    let unit: HKUnit?
+    var sensor: GlucoseDisplayableContext?
+    
+    let netBasal: NetBasalContext?
+    
+    let eventualGlucose: String?
+    
+    // For marking old entries as stale
+    var minsAgo: Int
+    
+    var isOld: Bool {
+        return minsAgo >= 5
+    }
+}
+
+struct SmallStatusWidgetEntryView : View {
+    var entry: Provider.Entry
+
+    var body: some View {
+        VStack(alignment: .center, spacing: 5) {
+            HStack(alignment: .center) {
+                LoopCircleView(entry: entry)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                // There is a SwiftUI bug which causes view not to be padded correctly when using .border
+                // Added padding to counteract the width of the border
+                    .padding(.leading, 8)
+                
+                GlucoseView(entry: entry)
+            }
+            .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .center)
+            .padding(5)
+            .background(
+                ContainerRelativeShape()
+                    .fill(Color("WidgetSecondaryBackground"))
+            )
+            
+            HStack(spacing: 2) {
+                if entry.isOld {
+                    Image(systemName: "arrow.clockwise")
+                        .font(.system(size: 10))
+                }
+                
+                Text("\(entry.minsAgo) min\(entry.minsAgo != 1 ? "s" : "") ago")
+                    .font(.caption)
+            }
+            .foregroundColor(.primary)
+            
+            
+            HStack(alignment: .center) {
+                BasalView(entry: entry)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                
+                VStack {
+                    if let eventualGlucose = entry.eventualGlucose {
+                        Text("Ev \(eventualGlucose)")
```

Comment:
"Ev" is too cryptic. Please update design to either say "Eventually", or "In x hours", or "Predicted". Something that conveys what this is, and add units.

Reply:
![IMG_1337E8FF71B4-1](https://user-images.githubusercontent.com/66573062/197410464-d1adbdae-2871-4ec9-a0aa-42b414524a13.jpeg)
Is this better?

Reply:
"Eventual" might be better, as that was a pre-existing term used in Loop. In any case, probably not all caps.

Reply:
Good point, I updated it.